api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -310,14 +310,17 @@ private Runtime getOverrideFromListRuntimes(String googleProject) {
   }
 
   @Override
-  public ResponseEntity<EmptyResponse> deleteRuntime(String workspaceNamespace) {
+  public ResponseEntity<EmptyResponse> deleteRuntime(
+      String workspaceNamespace, Boolean deleteDisk) {
     DbWorkspace dbWorkspace = lookupWorkspace(workspaceNamespace);
     String firecloudWorkspaceName = dbWorkspace.getFirecloudName();
     workspaceAuthService.enforceWorkspaceAccessLevel(
         workspaceNamespace, firecloudWorkspaceName, WorkspaceAccessLevel.WRITER);
 
     leonardoNotebooksClient.deleteRuntime(
-        dbWorkspace.getGoogleProject(), userProvider.get().getRuntimeName());
+        dbWorkspace.getGoogleProject(),
+        userProvider.get().getRuntimeName(),
+        deleteDisk != null && deleteDisk);
```

Comment:
nit: another way of writing this which might make the default value slightly more clear: `Optional.ofNullable(deleteDisk).orElse(false)`